framework/docs/src/10_integrations.md

```diff
@@ -0,0 +1,37 @@
+# Abstract Integrations
+
+!todo 
+
+- Here's a list of some of the projects we're currently integrated with:
```

Comment:
We should split this out into a big integrations section, and have them available as subsections on the sidebar. We did this similarly with [cw-orchestrator](https://orchestrator.abstract.money) and I feel like it draws users to click on the one they're interested in specifically.

Reply:
100%